recipes/minmea/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+from conan.tools.microsoft import is_msvc
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.53.0"
+
+class MinmeaConan(ConanFile):
+    name = "minmea"
+    description = "a lightweight GPS NMEA 0183 parser library in pure C"
+    license = "WTFPL"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kosma/minmea"
+    topics = ("gps", "NMEA", "parser")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    exports_sources = ["CMakeLists.txt"]
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if is_msvc(self):
+            raise ConanInvalidConfiguration(f"{self.ref} can not be built on Visual Studio and msvc.(yet)")
```

Comment:
What is breaking the build on MSVC?

Reply:
I missed this sentences in README.md.
https://github.com/kosma/minmea/blob/e368d847d75abd891c651f7d30ce5efb6681adb7/README.md#compatibility

I will try to support MSVC.